lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -16,10 +17,9 @@ def initialize(serializer, options = {})
         end
 
         def serializable_hash(options = nil)
-          options = {}
```

Comment:
Since it's critical to you, I'd think @joaomdmoura would support including that bugfix in this feature PR